docs/dashboards/kubernetes-dashboard.asciidoc

```diff
@@ -57,7 +57,12 @@ The DaemonSet integrates {elastic-endpoint} into your Kubernetes cluster. The {a
 
 You first need to download the DaemonSet manifest `.yaml`, then modify it to include your {fleet} URL and Enrollment Token before you deploy it to the clusters you want to monitor.
 
-1. Download the DaemonSet manifest using the command: `curl -L -O https://raw.githubusercontent.com/elastic/main/releases/8.4.0/kubernetes/deploy/elastic-endpoint-security.yaml`
+1. Download the DaemonSet manifest using the command: 
+[source,console]
+----
+curl -L -O https://raw.githubusercontent.com/elastic/endpoint/main/releases/8.4.0/kubernetes/deploy/elastic-endpoint-security.yaml
+----
+[start=2]
```

Comment:
Restarting the numbering isn't necessary here; we just need to make the source block a continuation of the step before it:
```suggestion
1. Download the DaemonSet manifest using the command: 
+
[source,console]
----
curl -L -O https://raw.githubusercontent.com/elastic/endpoint/main/releases/8.4.0/kubernetes/deploy/elastic-endpoint-security.yaml
----

```